README.rst

```diff
@@ -1,128 +1,94 @@
-
-*src/fortran/CMakeLists.txt* file::
-
-  # Define the executable and adding the source files required to build the application.
-  # When we are defining fortran source files if we name the source files with the extension
-  # .F90 (using a capital *F* and not a lowercase one) CMake will automatically run the *C*
-  # preprocessor on the file for us, saving us from specifying that through other means.
-  # This is only necessary if the file requires the the *C* preprocessor.
-  add_executable(XXXXXXXX ZZZZZZZZ.F90)
-  
-  # Set the required libraries for this executable.  Here we let CMake know which libraries
-  # are required to link the application.  Now because the *opencmisslibs* is known to CMake 
-  # as a target, there is nothing else we need to do.  The *opencmisslibs* target contains
-  # enough information for CMake to properly build and link the application with the OpenCMISS
-  # libraries.
-  #
-  # While https://github.com/OpenCMISS/iron/issues/88 is not fixed we also need to specify that
-  # MPI should also be linked into the application.  Once this issue is resolved we will no longer
-  # be required to add this as a link library.  This only applies to examples making use of Iron, 
-  # if the example is only using the Zinc library then MPI is not required at all.
-  target_link_libraries(XXXXXXXX PUBLIC opencmisslibs mpi)
-
-
-Inputs
-======
-
-If the example requries external inputs to be supplied these are stored in a directoy named *inputs*.  To specify the arguments required to run the executable write the arguments as a semi-colon separated list in a file named *arguments.cmake*.  The arguments specified in the *arguments.cmake* file and the inputs stored in the directory should match with the information stored in the *expected_results* section.  That is when the arguments taken from the *arguments.cmake* file are applied to the executable the output from the application should match what is in the *expected_results* directory to within some tolerance (when dealing with numerical values).
-
-The *inputs* directory should be made a sub-directory of the language.  For example in a C++ example we would have the following directory structure::
-
-    .
-    +-- docs
-    +-- src
-        +-- cxx
-            +-- inputs
-
-Expected results
-================
-
-If the example has some expected results these are stored in a directory named *expected_results*.  If the example writes text to the standard output stream then this content should be captured in a *stdout.txt* file within the *expected_results* directory.
-
-The *expected_results* directory should be made a sub-directory of the language.  For example in a Python example we would have the following directory structure::
-
-    .
-    +-- docs
-    +-- src
-        +-- python
-            +-- expected_results
-
-Documentation
+Python version::
+
+  source /path/to/opencmisslibs/install/virtaul_environments/oclibs_venv_pyXY_release/bin/activate
```

Comment:
I would prefer to see:

`source <path-to-opencmiss>/install/virtual_environments/oclibs_venv_pyXY_release/bin/activate`

where XY are the major, minor version numbers of Python respectively.

This would be consistent with other examples.